ci/install.sh

```diff
@@ -42,6 +45,17 @@ main() {
            --git japaric/cross \
            --tag $tag \
            --target $target
+
+    if [ $TRAVIS_OS_NAME = linux ]; then
+        echo "AM INSTALLING"
```

Comment:
Could these messages be more human readable: no all caps, full English sentences?

Reply:
This is only temporary and for me so I know what's going in logs